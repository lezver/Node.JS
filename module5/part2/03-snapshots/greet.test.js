const greet = require("./greet");

describe("greet", () =>
	it("shuld return 'Hellow World'", () => expect(greet()).toMatchSnapshot()));

describe("greet", () =>
	it("shuld return: Hellow World", () =>
		expect(greet()).toMatchInlineSnapshot(`"Hellow World"`)));
