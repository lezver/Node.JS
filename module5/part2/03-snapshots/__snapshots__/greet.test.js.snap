// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`greet shuld return 'Hellow World' 1`] = `"Hellow World"`;
